acceptance/testdata/auth/auth-switch.txtar

```diff
@@ -0,0 +1,58 @@
+# This requires a second account to log in to, meaning the GH_USER, 
```

Comment:
🤔 I'm wondering if there's a way to skip this test if any of the optional env vars aren't set... Or maybe we just skip it by default?

Reply:
I'm noodling on an idea like:

```
requireenv
```

or

```
skip-if-missing-env
```

These would only be accepted by default but in CI we would have a `GH_ACCEPTANCE_STRICT=true` that doesn't allow this, so that tests aren't skipped but fail. Or we could just assert no skipped tests. Just different tradeoffs.

Reply:
🤔 Different tradeoffs, for sure... I'd probably want to see how many more Env Vars we're gonna need to get more of the full test suite implemented to understand how flexible it needs to be. Definitely out of scope of this